documentation/localization_howto.md

```diff
@@ -0,0 +1,165 @@
+Localizing Streisand in your language
+=====================================
+
+One of the most unique aspects of the Streisand project are the detailed and customized
+instructions that it generates for each of the VPN services it provides, allowing them 
+to be easily shared among fellow users. They can help in removing ambiguity in how certain 
+services need to be set up and can be easily updated as the environment evolves.
+
+Scope
+-----
+
+The primary focus of localization efforts would be best concentrated on user facing
+content, as opposed to developer oriented content.
+
+For example, documentation that can help a user navigate the particularities of certain
+cloud providers (account setup, permissions, etc...) are good candidates for localization.
+
+On the other hand, developer oriented texts, such as Ansible task names aren't needed at
+this point.
+
+Language codes
+--------------
+
+For simplicity, it is preferable to use 2 character language codes (fr -> French, ar -> Arabic),
+as defined by [ISO 631-1](https://en.wikipedia.org/wiki/ISO_639). For certain locales such as
+Chinese vs Simplified Chinese, 3 character codes are also acceptable.
+
+Filename conventions
+--------------------
+
+In order to differentiate a localized file, append the lowercase language code after the
+filename with a 'dash' symbol (e.x. README-fr.md for a README localized in French).
+
+Point of references
+-------------------
+
+When translating, English should be considered the point of reference to translate *from*.
+
+READMEs and Documentation
+-------------------------
+
+The first interaction with Streisand for an end user would be first visiting the README,
+where all the necessary steps one would need to set up their computing environment are
+detailed, as well as a listing of all the VPN services provided by Streisand are listed.
+
+Unlike Streisand's instructions markdown template files, READMEs and documentation are
+orthogonal to Streisand being able to successfully build. As a consequences, it is possible
+to have READMEs and documentation in more languages than Streisand builds with.
```

Comment:
maybe disambiguate slightly by saying "in more languages than Streisand builds documentation with"